src/proto/Cosmos.proto

```diff
@@ -119,6 +119,14 @@ message Message {
         repeated string coin = 6;
     }
 
+    // thorchain/MsgSend
+    message THORChainSend {
+        bytes from_address = 1;
+        bytes to_address = 2;
+        repeated Amount amounts = 3;
+        string type_prefix = 4;
```

Comment:
Does it need? I think we can hardcode for thorchain

Reply:
Ok.  Prefix is not needed for protobuf version, anyhow.